CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## 16.13.0 - 2025-01-07
+
+It adds 3 rules to help you write error-free at-rules and 2 rules to warn you about deprecated CSS features. We've turned these rules on in our [standard config](https://www.npmjs.com/package/stylelint-config-standard).
+
+It also adds new rule options, a feature to time how long rules take, lax autofix and support for `messageArgs` in more rules. It fixes 7 bugs.
```

Comment:
nit

The summary looks good as is but you could maybe reword that part

> a feature to time how long rules take

Maybe use the word _display_ or _track_?
cf https://eslint.org/docs/latest/extend/custom-rules#profile-rule-performance

> It fixes 7 bugs.

Are you counting the PRs with a Fix title or the issues closed of type _Bug_ or the `- Fixed: …` lines?

Reply:
Kept it simple and counted the number of lines.